environment.py

```diff
@@ -616,6 +616,13 @@ def env() -> Mapping[str, Optional[str]]:
         #
         'AZUL_ENABLE_MONITORING': '0',
 
+        # Enable populating relations in the verbatim PFB export (1 yes, 0 no)
+        # Note that a given plugin may not support the verbatim PFB manifest
+        # format, or may not support adding relations if it does, in which case
+        # this variable has no effect.
+        #
+        'AZUL_ENABLE_VERBATIM_RELATIONS_EXPORT': '1',
```

Comment:
The variable name is a bit unwieldy. "Export" is not Azul lingo. The deleted part is obvious, IMO. 

```suggestion
        # Whether to include relations in manifests of the verbatim PFB format (1 yes, 0 no)
        #
        'AZUL_ENABLE_VERBATIM_RELATIONS': '1',
```